lib/resty/openssl.lua

```diff
@@ -448,4 +448,11 @@ function _M.list_ssl_ciphers(cipher_list, ciphersuites, protocol)
   return ssl_lib.get_ciphers(ssl)
 end
 
+function _M.get_server_certificate(ssl_ptr)
```

Comment:
We might want to use this route https://github.com/fffonion/lua-resty-openssl/commit/48c51077444e375b2cdd5155693b49d92a82d4a1 and add teh get_request_socket_ssl into it. It will be bit un-user-friendly to require
user to pass a raw SSL pointer to be able to use this function.